core/Command/User/Profile.php

```diff
@@ -0,0 +1,230 @@
+			)
+			->addOption(
+				'update-only',
+				null,
+				InputOption::VALUE_NONE,
+				'Only updates the value, if it is not set before, it is not being added'
+			)
+
+			// Delete
+			->addOption(
+				'delete',
+				null,
+				InputOption::VALUE_NONE,
+				'Specify this option to delete the property value'
+			)
+			->addOption(
+				'error-if-not-exists',
+				null,
+				InputOption::VALUE_NONE,
+				'Checks whether the property exists before deleting it'
+			)
+		;
+	}
+
+	protected function checkInput(InputInterface $input) {
+		$uid = $input->getArgument('uid');
+		$user = $this->userManager->get($uid);
+		if (!$user) {
+			throw new \InvalidArgumentException('The user "' . $uid . '" does not exist.');
+		}
+		// normalize uid
+		$input->setArgument('uid', $user->getUID());
+
+		if ($input->getArgument('key') === '' && $input->hasParameterOption('--default-value')) {
+			throw new \InvalidArgumentException('The "default-value" option can only be used when specifying a key.');
+		}
+
+		if ($input->getArgument('key') === '' && $input->getArgument('value') !== null) {
+			throw new \InvalidArgumentException('The value argument can only be used when specifying a key.');
+		}
+		if ($input->getArgument('value') !== null && $input->hasParameterOption('--default-value')) {
+			throw new \InvalidArgumentException('The value argument can not be used together with "default-value".');
+		}
+		if ($input->getOption('update-only') && $input->getArgument('value') === null) {
+			throw new \InvalidArgumentException('The "update-only" option can only be used together with "value".');
+		}
+
+		if ($input->getArgument('key') === '' && $input->getOption('delete')) {
+			throw new \InvalidArgumentException('The "delete" option can only be used when specifying a key.');
+		}
+		if ($input->getOption('delete') && $input->hasParameterOption('--default-value')) {
+			throw new \InvalidArgumentException('The "delete" option can not be used together with "default-value".');
+		}
+		if ($input->getOption('delete') && $input->getArgument('value') !== null) {
+			throw new \InvalidArgumentException('The "delete" option can not be used together with "value".');
+		}
+		if ($input->getOption('error-if-not-exists') && !$input->getOption('delete')) {
+			throw new \InvalidArgumentException('The "error-if-not-exists" option can only be used together with "delete".');
+		}
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output): int {
+		try {
+			$this->checkInput($input);
+		} catch (\InvalidArgumentException $e) {
+			$output->writeln('<error>' . $e->getMessage() . '</error>');
+			return 1;
+		}
+
+		$uid = $input->getArgument('uid');
+		$key = $input->getArgument('key');
+		$user = $this->userManager->get($uid);
+		$userAccount = $this->accountManager->getAccount($user);
+
+		if ($key === '') {
+			$settings = $this->getAllProfileProperties($userAccount);
+			$this->writeArrayInOutputFormat($input, $output, $settings);
+			return 0;
```

Comment:
```suggestion
			return self::SUCCESS;
```
Same change on all returns